### ISSUE TITLE
Custom fields

### ISSUE THREAD

**Issue by abeumer** [tessus/mwExtensionMantis#12]
I've added custom fields to our MantisBT installation (1.3.1). Is it possible to include them in the table when specifying the show parameter on a wiki page?

**Comment by tessus** [comment]
No, it is not possible at the moment.

But I will look into it when I get some time.

**Comment by abeumer** [comment]
I read that you did not receiving any feedback on a similar question in the past. So when you do have time, just let me know and I will provide you with information if I can

**Comment by tessus** [comment]
Thank you for the offer. I will upgrade my Mantis installation to 1.3.1 and look into custom fields.
Maybe you can give me a quick and easy example for a custom field, how to create it, and use it.

I suspect that custom fields are stored in a separate table, which means I will have to join an additional table in the query. But I will also have to investigate, if indexes are properly set, otherwise this can an have a negative performance impact with large data sets.

But your request is a good idea and this feature will be a nice addition to the extension.

**Comment by abeumer** [comment]
Hi Tessus,

`mantis_custom_field_table`
In our case I've added a custom field "notify customer"  which is a basic yes/no field. We use it to mark the need to notify end-users about the error.

(I've translated the values to English)
INSERT INTO `mantis_custom_field_table` (`id`, `name`, `type`, `possible_values`, `default_value`, `valid_regexp`, `access_level_r`, `access_level_rw`, `length_min`, `length_max`, `require_report`, `require_update`, `display_report`, `display_update`, `require_resolved`, `display_resolved`, `display_closed`, `require_closed`, `filter_by`) VALUES
(1, 'Notify customer', 9, 'Yes|No', 'No', '', 10, 25, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 1);

By the look of it, it seems that the custom fields are stored in that table but their assignment to a project is stored in `mantis_custom_field_project_table` and when the value is set it seems to be stored in `mantis_custom_field_string_table`.

**Comment by tessus** [comment]
Current status: I hit a snag right now, because my development system is down. I hope I will be able to work on it again soon.

**Comment by abeumer** [comment]
Any luck with the development system?

**Comment by tessus** [comment]
@abeumer yes, I was able to get a dev system back up, but after looking into this issue, I have sincere doubts. the code changes would be extensive and would also have a negative effect on performance.
Until I find a really good way to make this work, I'll put this on hold.
